src/adapters/cmsClient.ts

```diff
@@ -1,21 +1,25 @@
 import { captureException } from '@sentry/nextjs'
 
+const isProduction = process.env.NEXT_PUBLIC_ENV === 'production'
```

Comment:
Purée si j'étais salé je te lacherai un : "not related"

Reply:
C'est totalement relié !!! On utilisait indifféremment `NODE_ENV` et `NEXT_PUBLIC_ENV`. C'est pas la même chose. Sur une RA `NODE_ENV=production` `NEXT_PUBLIC_ENV=pre-production`

Reply:
Ahhhhh hihi ça roule